### ISSUE TITLE
Use stacrs to read+write stac-geoparquet

### ISSUE THREAD

**Issue by gadomski** [stac-utils/pystac#1455]
**stacrs** is a zero-dependency Python package that can read and write [stac-geoparquet](https://github.com/stac-utils/stac-geoparquet) files. We could add **stacrs** as an optional dependency here to enable **stac-geoparquet** for **pystac**.

The option of adding **stacrs** to **pystac** was discussed in this issue: https://github.com/stac-utils/stac-rs/issues/462

> [!NOTE]
> I am the author of **stacrs** so I may have some bias.

**Comment by jsignell** [comment]
Does it only make sense to read-write `ItemCollection`s as stac-geoparquet?

**Comment by gadomski** [comment]
I think so. Right now we're not sure exactly how to handle collections in the geoparquet metadata (https://github.com/stac-utils/stac-rs/issues/428, I really should open up an issue on the spec for that one).

**Comment by jsignell** [comment]
Yeah I was just looking at that because I was thinking about #1254 and noticed that the geoparquet that comes out of stacrs right now does not have the collection-level fields at the top.
